layouts/millennium-overview.html

```diff
@@ -0,0 +1,27 @@
+<%= render '/head.*' %>
+
+<body class="<%= @item[:version].to_s %>_api">
```

Comment:
this generates the following html

```
<body class="_api" style="">
```

compared to e.g. the api layout which generates this html

```
<body class="dstu2_api" style="">

or

<body class="r4_api" style="">
```

as far as i can tell, these classes aren't directly used for any sort of styling. so in this case, just put a plain body tag

Reply:
Removed here: https://github.com/cerner/fhir.cerner.com/pull/396/commits/41817e620cb4f4169f1159cc0a7c4b08a75396a7